### ISSUE TITLE
Wildcard paths not working properly for file watcher

### ISSUE THREAD

**Issue by binaryfire** [laravel/octane#962]
### Octane Version

2.5.9

### Laravel Version

11

### PHP Version

8.3

### What server type are you using?

Swoole

### Server Version

8.3

### Description

Wildcard paths like `config/**/*.php` in the `watch` array of `config/octane.php` aren't working properly.

### Steps To Reproduce

1. Install chokidar
2. Run `octane:start --watch` (I'm using Swoole)
3. Modify a file in the `config` directory. Octane won't restart
4. Change `config/**/*.php` to `config` in the `watch` array of `config/octane.php`
5. Restart Octane server
6. Modify a file in the `config` directory. Octane will now correctly restart

Same problem with anything else using wildcards eg. `database` (try modifying a seeder to test).

**Comment by crynobone** [comment]
Shouldn't it be `config/*.php`?

**Comment by binaryfire** [comment]
@crynobone `config/**/*.php` is what's in the default config file: https://github.com/laravel/octane/blob/46a84995edb8b4334352edf071eb36e4797cba41/config/octane.php#L189

But it should work either way. The `**` pattern is supposed to match files in the root dir and all subdirs. It's probably more important for things like the public folder (`public/**/*.php'`). But as a side note, I seem to remember reading that config subdirectories are supported by the framework using dot notation.

The problem is reproducible with the default Octane config - you can test using any other directory in the array which uses `**` in the path.

**Comment by binaryfire** [comment]
@crynobone Yeah looks config subdirs are supported, so `config/**/*.php` is the correct expression: https://laracasts.com/discuss/channels/laravel/laravel-8-put-config-files-in-a-subdirectory-of-config

**Comment by ChrisExP** [comment]
Chokidar removed support for glob in version 4.
They give suggestions for how to reproduce the functionality, but I don't know how that should work for Laravel Octane.
You can get it working by installing version 3 instead.

https://github.com/paulmillr/chokidar?tab=readme-ov-file#changelog

**Comment by crynobone** [comment]
Fixed via https://github.com/laravel/octane/releases/tag/v2.8.2
